src/lib.rs

```diff
@@ -40,7 +40,9 @@ use log::Log;
 use log::*;
 
 #[cfg(feature = "paris")]
-mod paris;
+mod paris_macros;
+#[cfg(feature = "paris")]
+pub extern crate paris;
```

Comment:
Oh sorry, this is not what I meant. Exporting the `paris` crate does not expose our new macros.
We need to export our `paris` **module** (now named `paris_macros`). :sweat_smile:

Reply:
updated ... I hope it is now ok :)

Reply:
bump :)